macos/FluentUI/AvatarView/AvatarView.swift

```diff
@@ -480,3 +531,26 @@ fileprivate extension CGPath {
 		return CGPath(ellipseIn: NSRect(x: 0, y: 0, width: diameter, height: diameter), transform: nil)
 	}
 }
+
+fileprivate extension NSColor {
+	enum InterfaceStyle: String {
+	   case dark, light
+
+		init() {
+			var type: InterfaceStyle = .light
+			let isDarkMode = NSApplication.shared.effectiveAppearance.bestMatch(from: [.darkAqua]) == .darkAqua
+			if isDarkMode {
+				type = .dark
+			}
+			self = type
+		}
+	}
+
+	static func getColor(light: NSColor, dark: NSColor) -> NSColor {
+		let currentStyle = InterfaceStyle()
+		if currentStyle == .light {
+			return light
+		}
+		return dark
+	}
```

Comment:
I think we very much don't like things like "isDarkMode" boolean from my memory of when we added Dark Mode to office, but I can't remember specific reasons. Is there any reason we can't just pull the asset catalog color and let the system pull the right color?

Reply:
dynamic traits are not supported by [NSColor](https://developer.apple.com/documentation/appkit/nscolor/color_creation) creation API's. 

Hopefully, Apple will have an addition where we can load them dynamically as opposed to using memory values.

Reply:
@heyimchris do you remember (from the Canvas UX days of Dark Mode integration) whether this might cause any issues? I would ask Mark but he is on parental leave.

Reply:
I replaced this call with reading from `userDefaults`. The notification when `effectiveApperance` is changed doesn't provide information about theme. If there are alternatives suggestion please feel free to create a new GitHub issue or let me know.